nodejs/control

```diff
@@ -0,0 +1,16 @@
+Package: nodejs
```

Comment:
How does this part work? Is this information necessary for sbom creation? I'm worried about the "maintainer" aspect.

Reply:
we could change the maintainer fields. I took it from nodesources deb to have similar control files. this file is used by both sbom and dpkg. sbom converts this file to spdx and dpkg just puts it into its respective location.

Reply:
I might just point it to nodejs.org instead of distroless

Reply:
i can commit the changes if you could provide a diff here.

Reply:
friendly ping